.scalafmt.conf

```diff
@@ -1,7 +1,12 @@
-version = "1.5.1"
-
 project.git = true
 
-align = none # never align to make the fmt more diff friendly
+align.preset = none # never align to make the fmt more diff friendly
 maxColumn = 100
 runner.fatalWarnings = true
+danglingParentheses.defnSite = false
+danglingParentheses.callSite = false
+trailingCommas = multiple
+newlines.implicitParamListModifierPrefer = before
+newlines.avoidAfterYield = false
+newlines.beforeCurlyLambdaParams = multilineWithCaseOnly
+docstrings.style = SpaceAsterisk
```

Comment:
Which of these do you consider "good" and which do you consider "bad, but what we currently do"?

Reply:
I definitely consider trailingCommas good, tbh I don’t care about the rest. I use formatters to avoid wasting time on bikeshedding not because I care about a particular formatting.

Reply:
Cool, sounds good. I think I'd prefer the `danglingParentheses` rules were `true` for the same reason as `trailingCommas`: smaller diffs. Ambivalent about the rest, so I'd probably want to just go with whatever the defaults are unless we have a good reason.

Reply:
alright, I set everything to the default which includes danglingparentheses and only enabled trailingcommas and no alignment.

Reply:
it does change the diff from 7k lines to 24k lines though

Reply:
Yeah, I didn't mean you should change them now, just that I'd like to get there at some point. :slightly_smiling_face:

Reply:
well 2 people have approved it now and nobody has complained so I guess I might as well do it in one go.